test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationDomainRepositoryTests.cs

```diff
@@ -383,4 +385,329 @@ public async Task GetVerifiedDomainsByOrganizationIdsAsync_ShouldVerifiedDomains
         Assert.Null(otherOrganizationDomain);
         Assert.Null(unverifiedDomain);
     }
+
+    [DatabaseTheory, DatabaseData]
```

Comment:
`DatabaseTheory` is deprecated, you can just use the regular `Theory` attribute.